gst/nnstreamer/nnstreamer_plugin_api_util_impl.c

```diff
@@ -422,17 +422,21 @@ gst_tensors_info_free (GstTensorsInfo * info)
 gsize
 gst_tensors_info_get_size (const GstTensorsInfo * info, gint index)
 {
+  GstTensorInfo *_info;
   gsize data_size = 0;
   guint i;
 
   g_return_val_if_fail (info != NULL, 0);
   g_return_val_if_fail (index < (gint) info->num_tensors, 0);
 
   if (index < 0) {
```

Comment:
How about fixing it with ```if (index == -1)```?
The documentation only mentions -1. (* @param index the index of tensor (-1 to get total size of tensors))

Reply:
supposed negative index value to get total tensor size in the past.

Reply:
Okay, Thank you.